iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -180,6 +244,35 @@ TEST_F(RoudiFindService_test, StopOfferMultiMethodServiceSingleInstance)
     receiverRuntime->findService({"service3", "instance1"}, instanceContainer);
     ASSERT_THAT(instanceContainer.size(), Eq(0u));
 }
+
+TEST_F(RoudiFindService_test, StopOfferServiceSingleMultipleCalls)
+{
+    senderRuntime->offerService({"service1", "instance1"});
+    this->InterOpWait();
+    senderRuntime->stopOfferService({"service1", "instance1"});
+    this->InterOpWait();
+    senderRuntime->stopOfferService({"service1", "instance1"});
+    this->InterOpWait();
+
+    iox::runtime::InstanceContainer instanceContainer;
+    receiverRuntime->findService({"service1", "instance1"}, instanceContainer);
+    ASSERT_THAT(instanceContainer.size(), Eq(0u));
+}
+
+
+TEST_F(RoudiFindService_test, StopNonExistingService)
```

Comment:
I think the test here is to verify that stopping a non existing service does not stop an existing service?!

Reply:
test is to check stopping non existing services will silently ignore the request, additionally check was introduced to confirm that it does not stop existing service